empty/bin/www

```diff
@@ -34,7 +34,7 @@ server.on('listening', onListening);
  */
 
 function normalizePort(val) {
-  var port = parseInt(val, 10);
+  let port = parseInt(val, 10);
```

Comment:
This could probably be `const` as well - I don't imagine the `port` would change?  Most of the `let`s in this file could probably be `const`s